AsyncDisplayKit/ASCollectionView.h

```diff
@@ -171,6 +171,13 @@ NS_ASSUME_NONNULL_BEGIN
 - (void)relayoutItems;
 
 /**
+ * Triggers a recalulation of leadingScreensForBatching
+ * @discussion During various layout changes that trigger content size changes, batch fetching needs to recalulate it's number of screens left to scroll before the delegate -collectionView:beginBatchFetchingWithContext: is called. Call when a layout transition is complete, or it will have no effect.
+ *
+ */
+- (void)invalidateBatchFetchMeasurements;
```

Comment:
@appleguy All that said I think the _correct_ way to go is ultimately decoupling scroll from this as a hard dependency, and making invalidation/re-measurement of batch trigger in a more generic way. That would solve it without exposing anything. Fear we may be a bit far away from that currently though, from a ASDK priority standpoint (I'm guessing this of course)..